catroid/src/main/java/org/catrobat/catroid/ui/WebViewActivity.java

```diff
@@ -194,6 +194,13 @@ public void onPageStarted(WebView view, String urlClient, Bitmap favicon) {
 					} catch (ClassNotFoundException e) {
 						Log.e(TAG, e.getMessage());
 					}
+				} else if (BuildConfig.PHIRO_CODE) {
```

Comment:
same here, why not use flavored classes to do this instead of Class.forName based on a build flag?